README.md

```diff
@@ -75,3 +73,27 @@ This command only works if:
 
 - you are in a local Git repository
 - the repository has a file called ‘.pre-commit-config.yaml’
+
+``` bash
+spaid_create_function_file -h
```

Comment:
Hmm, could it be 
```suggestion
spaid_create_python_file -h
```
instead? Just thinking with e.g., the properties example below that the file doesn't actually contain functions - it contains classes.

Reply:
Good point!